samples/solidity/hardhat.config.ts

```diff
@@ -22,8 +22,13 @@ const config: HardhatUserConfig = {
       },
     },
   },
-  defaultNetwork: "firefly",
+  defaultNetwork: "mumbai",
```

Comment:
Do we really want to change the default for `npm run deploy`?

Reply:
Also didn't mean to check this in...